src/codewhisperer/util/commonUtil.ts

```diff
@@ -27,3 +32,27 @@ export function isAwsError(error: any): boolean {
         error.time instanceof Date
     )
 }
+
+export async function getDirSize(dirPath: string, startTime: number, duration: number): Promise<number> {
```

Comment:
I was wondering if it would make sense to exclude the build artifacts (or binary files) in the project from the project size as build artifacts can be very large compared to the source files and I think we are interested in knowing what percentage of the source files were actually scanned. Including the build artifacts might skew this data.

Reply:
In that case we probably want to re-use the dependency graph logic for source file detection here. I'd also update [the description](https://github.com/aws/aws-toolkit-common/commit/e7cfb5668f0e01431e643b5155f25bf6eea4f40e#diff-f9ad596fbedec6dc08aa26f65b8a0f9358ae2be13476b00a82557f7d7125c568R408) to clarify that only source files should be included.